docs/OpenCLCOnVulkan.md

```diff
@@ -161,10 +161,10 @@ Consider this example:
 
 It generates the following descriptor map:
 
-    kernel,foo,arg,a,argOrdinal,0,descriptorSet,0,binding,0,offset,0,argKind,buffer
-    kernel,foo,arg,f,argOrdinal,1,descriptorSet,0,binding,1,offset,0,argKind,pod
-    kernel,foo,arg,b,argOrdinal,2,descriptorSet,0,binding,2,offset,0,argKind,buffer
-    kernel,foo,arg,c,argOrdinal,3,descriptorSet,0,binding,3,offset,0,argKind,pod
+    kernel,foo,arg,a,argOrdinal,0,descriptorSet,0,binding,0,offset,0,argSize,0,argKind,buffer
+    kernel,foo,arg,f,argOrdinal,1,descriptorSet,0,binding,1,offset,0,argSize,4,argKind,pod
```

Comment:
As per @ericberdahl 's commentary, I'd like this only on pod and pod_ubo.
And put the new argSize after  the argKind key/value pair.

Reply:
Done.